README.md

```diff
@@ -31,6 +31,66 @@ And below are the lower-level packages, used internally, or by advanced users.
 | [`@parity/contracts`](/packages/contracts) | [![npm (scoped)](https://img.shields.io/npm/v/@parity/contracts.svg)](https://www.npmjs.com/package/@parity/contracts) | [![README](https://img.shields.io/badge/docs-README-green.svg)](/packages/contracts#readme)      | Parity's [contracts](https://github.com/parity-contracts) as ES6 classes. |
 | [`@parity/electron`](/packages/electron)   | [![npm (scoped)](https://img.shields.io/npm/v/@parity/electron.svg)](https://www.npmjs.com/package/@parity/electron)   | [![README](https://img.shields.io/badge/docs-README-green.svg)](/packages/electron#readme)       | Control the Parity Ethereum node from Electron.                           |
 
+### Contributing
+
+#### Dependencies
+
+Install at least `yarn` version 1.4.2 and [Node.js >=10.10.0](https://nodejs.org/en/)
+
+```
+yarn --version // Should be at least 1.4.2
+```
+
+#### Tests
+
+```
+yarn test
+```
+
+#### Build
+
+```
+yarn build
+```
+
+#### Maintenance
+
+1. Fork the repo
+
+2. Clone your fork
+```
+git clone https://github.com/<INSERT_YOUR_GITHUB_USERNAME>/js-libs
+```
+
+3. Check outdated dependencies
+
+```
+yarn outdated
+```
+
+4. Create a branch
+
+```
+git checkout -b <INSERT_YOUR_BRANCH_NAME>
+```
+
+5. Run tests, linting, and build
+
+```
+yarn test; yarn lint; yarn build
+```
+
+6. Push the branch to your fork of the repo
+
+7. Integrate the updated library as a dependency. Example: If you are using the 'electron' package in another project, then update the package.json file to temporarily use your branch instead of the public NPM registry version
```

Comment:
@ltfschoen I understand now where your errors come from. I would like to remove this step 7: For `@parity/*` packages, I would advise against doing `yarn add github:something` into your own project.

The reason is:
- right now we develop in `src/`
- the CI builds, outputs in `lib/`, and pushes `lib/` to npm
- when you `yarn add @parity/*` it downloads `lib/` from npm, and the main file is `lib/index.js`

However, when you `yarn add github:something`, it does:
- clone the repo
- run yarn build inside the cloned folder
- hopefully, if the build doesn't error, it generates a `lib/` folder 

Jaco and I used to try to do this method above, however there's always some troubles here and there in the "hopefully" step. That's why I prefer to remove it. Some alternative solutions:
- (ideal solution) wait for the npm package to be published. If you need to test a package, it's better to add a test directly in this repo, so that the test is self-contained.
- use npm link
- do the copy/paste dance:
```
# in js-libs/
yarn build
cp -r packages/my-package/lib ../fether/node_modules/@parity/my-package/lib
# in fether
yarn start
```